src/server.ts

```diff
@@ -0,0 +1,29 @@
+import express from "express";
+import axios from "axios";
+
+import { getRovers, getRoverPhotos } from './Controllers/controller';
+
+const app = express();
+const port = 8000;
+
+app.use(express.json());
+const router = express.Router();
+
+
+router.get('/test', (req, res) => res.send('Hello world !'));
+app.use('/', router);
+
+
+app.get("/rovers", async (req, res) => {
+    res.send(await getRovers());
+});
+
+app.get("/rovers/:rover_name/photos/:camera_type/:sol/:page", async (req, res) => {

```

Comment:
the controller contains the callbacks. see https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes#create_the_route-handler_callback_functions